src/js/letters/containers/Main.jsx

```diff
@@ -14,29 +16,45 @@ class Main extends React.Component {
   render() {
     let appContent;
 
-    if (this.props.lettersAvailability === 'available') {
-      appContent = this.props.children;
-    } else if (this.props.lettersAvailability === 'awaitingResponse') {
-      appContent = <LoadingIndicator message="Loading your letters..."/>;
-    } else if (this.props.lettersAvailability === 'unavailable') {
-      appContent = (
-        <div>
-          <div className="usa-alert usa-alert-error" role="alert">
-            <div className="usa-alert-body">
-              <h4 className="usa-alert-heading">Letters Unavailable</h4>
-              <p className="usa-alert-text">
-                We weren't able to retrieve your VA letters. Please call
-                1-855-574-7286 between Monday-Friday 8:00 a.m. - 8:00 p.m. (ET).
-              </p>
+    switch (this.props.lettersAvailability) {
+      case 'available':
```

Comment:
Does it make sense for these strings to be shared constants? They're used in a couple places, it looks like.